roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -30,13 +30,19 @@ def get_info_from_hammer(command, column=1)
     external_capsules << {:id => id, :name => name, :lifecycle_environments => lifecycle_environment}
   end
 
-
   reverse_commands = []
+  satellite_version = ARGV[0].to_f
   external_capsules.each do |capsule|
     capsule[:lifecycle_environments].each do |env|
-      run_hammer_cmd("--csv capsule content remove-lifecycle-environment --id #{capsule[:id]} --environment-id #{env}")
-      reverse_command = prepare_hammer_cmd("--csv capsule content add-lifecycle-environment --id #{capsule[:id]} --environment-id #{env}")
-      reverse_commands << reverse_command
+      if satellite_version > 6.5
+        run_hammer_cmd("--csv capsule content remove-lifecycle-environment --id #{capsule[:id]} --lifecycle-environment-id #{env}")
```

Comment:
Can you not repeat `--csv capsule content remove-lifecycle-environment --id #{capsule[:id]} ` and `--csv capsule content add-lifecycle-environment --id #{capsule[:id]}`

You could define something like this:
```ruby
def capsule_lce_args(action, capsule_id, env, lifecycle=false)
  "--csv capsule content #{action}-lifecycle-environment --id #{capsule_id} --#{lifecycle ? "lifecycle-" : ""}environment-id #{env}"
end
```

and could use in the following formats:
```ruby
run_hammer_cmd(capsule_lce_args("remove", capsule[:id], env, true)) # 6.5+
prepare_hammer_cmd(capsule_lce_args("add", capsule[:id], env, true)) # 6.5+
run_hammer_cmd(capsule_lce_args("remove", capsule[:id], env, false)) # < 6.5
prepare_hammer_cmd(capsule_lce_args("add", capsule[:id], env, false)) # 6.5+
```

Reply:
Fixed